terra-jupyter-python/Dockerfile

```diff
@@ -73,7 +77,7 @@ RUN pip3 -V \
  && pip3 install intel-openmp==2018.0.0 \
  && pip3 install mkl==2018.0.3 \
  && pip3 install readline==6.2 \
- && pip3 install setuptools==36.4.0 \
+ && pip3 install setuptools==42.0.2 \
```

Comment:
six and setuptools are upgraded due to some google-cloud-xxx lib requires higher version